<!-- -->
<launch>
    <arg name='host' default='localhost'/>
    <arg name='port' default='2000'/>
    <arg name='number_of_pedestrians' default='10'/>
    <arg name="role_name" default="ego_pedestrian"/>
    <param name="carla/host" value="$(arg host)" unless="$(eval host == '')"/>
    <param name="carla/port" value="$(arg port)" unless="$(eval port == '')"/>
    <param name="carla/pedestrians" value="$(arg number_of_pedestrians)" unless="$(eval port == '')"/>
    <node pkg="carla_ros_spawn_npc" type="carla_ros_spawn_pedestrians.py" name="carla_ros_spawn_npc" output="screen">
    </node>
</launch>